projectlicense.md

```diff
@@ -1,61 +1,51 @@
 # Licensing terms
 
-This project is licensed under the terms of the Modified BSD License
-(also known as New or Revised or 3-Clause BSD), as follows:
-
-- Copyright (c) 2001-2015, IPython Development Team
-- Copyright (c) 2015-, Jupyter Development Team
+This project is licensed under the terms of the 3-Clause BSD License (also
+known as New or Revised or Modified BSD License, or the BSD-3-Clause license),
+as follows:
 
+Copyright (c) YEAR Project Jupyter Contributors.
```

Comment:
should `YEAR` here remain a placeholder, or do we put 2015, cribbing from the proposed deletion?

Reply:
Good question. I figured the year would change depending on when that specific repo was started, so I left it a placeholder.